### ISSUE TITLE
Code in docs highlighted as Ruby

### ISSUE THREAD

**Issue by dblock** [falood/maru#25]
example: https://maru.readme.io/docs/unit-test

![screen shot 2016-06-08 at 1 38 07 pm](https://cloud.githubusercontent.com/assets/542335/15904267/4aceb296-2d7e-11e6-92ed-80cbc4a60ca5.png)

But it's not Ruby :)

**Comment by falood** [comment]
I deploy these docs on `readme.io` and `elixir` is not supported.

**Comment by dblock** [comment]
I don't think you should close issues that are real, other people will stumble upon them and maybe add their +1. Eventually there will be millions of people wanting Elixir support on readme.io or you'll want to move it to a different site.

**Comment by falood** [comment]
Thank you! I see.

**Comment by mikaoelitiana** [comment]
Maybe adding a tag will help understand it's not depending on you. I often see tags like waiting or something similar
